modules/trusted-firmware-m/Kconfig

```diff
@@ -305,4 +305,13 @@ config TFM_FLASH_MERGED_BINARY
 		(if building with TFM_BL2 is enabled), and the
 		Non-Secure application firmware.
 
+config TFM_EXTRA_CMAKE_ARGS
+	string
```

Comment:
I'm getting an error:
```
error: TFM_EXTRA_CMAKE_ARGS (defined at modules/trusted-firmware-m/Kconfig:308, modules/trusted-
firmware-m/Kconfig:308) is assigned in a configuration file, but is not directly user-configurable
(has no prompt). It gets its value indirectly from other symbols
```
Don't we need a prompt here?

Reply:
Same thing, adding `CONFIG_TFM_EXTRA_CMAKE_ARGS="-DFOO=bar"` to prj.conf in `psa_level_1` and building with `west build -p -b mps2_an521_nonsecure zephyr/samples/tfm_integration/psa_level_1`:

```
error: TFM_EXTRA_CMAKE_ARGS (defined at modules/trusted-firmware-m/Kconfig:308, modules/trusted-
firmware-m/Kconfig:308) is assigned in a configuration file, but is not directly user-configurable
(has no prompt). It gets its value indirectly from other symbols. See
http://docs.zephyrproject.org/latest/reference/kconfig/CONFIG_TFM_EXTRA_CMAKE_ARGS.html and/or look
up TFM_EXTRA_CMAKE_ARGS in the menuconfig/guiconfig interface. The Application Development Primer,
Setting Configuration Values, and Kconfig - Tips and Best Practices sections of the manual might be
helpful too.
```